apps/shipments/tasks.py

```diff
@@ -46,3 +46,11 @@ def telemetry_data_update(self, shipment_id, payload):
                             rate_limit=shipment.background_data_hash_interval,
                             action_type=AsyncActionType.TELEMETRY,
                             use_updated_by=False)
+
+
+@shared_task(bind=True)
+def gtx_validation_task(self, shipment_id):
+    log_metric('transmission.info', tags={'method': 'shipments_tasks.gtx_validation', 'module': __name__})
+    shipment = Shipment.objects.get(id=shipment_id)
+    shipment.validate_gtx()
```

Comment:
You can still add unit tests that test shipment.validate_gtx() without having to deal with testing the celery task directly

Reply:
I was running into an issue where when I tried to run shipment.validate_gtx and then shipment.save, it was causing an issue with protected fields when refreshing from the database

Reply:
Turns out I needed to only refresh the gtx_validation field on the shipment, readding the tests now.